crowbar_framework/app/constraints/api_constraint.rb

```diff
@@ -15,13 +15,25 @@
 #
 
 class ApiConstraint
-  attr_reader :version
+  attr_reader :versions
 
-  def initialize(options)
-    @version = options.fetch(:version)
+  def initialize(*versions)
+    @versions = versions
   end
 
   def matches?(request)
-    request.headers.fetch(:accept) == "application/vnd.crowbar.v#{version}+json"
+    allowed = false
+    major, minor = nil
+    versions.each do |version|
```

Comment:
`versions.any?` is slightly better — no explicit return variable, and also won't check next `version` values once the match is there.